### ISSUE TITLE
typeshare - does it create kotlin methods?

### ISSUE THREAD

**Issue by arabdulazeez** [1Password/typeshare#138]
I see that typeshare creates classes in kotlin for the equivalent rust classes but when it comes to creating methods, the library doesn't support it.
Any idea why that would be the case?

**Comment by czocher** [comment]
I don't think this can be done in most cases. For simple cases like getters/setters yes, but if a method has any logic translating this logic to all the various languages that typeshare produces wouldn't be possible without in-depth abstract syntax analysis on the Rust side.
I strongly suspect this is not in the roadmap for this library and it will never be implemented.
